Doc/faq/design.rst

```diff
@@ -232,7 +232,7 @@ Similar methods exist for bytes and bytearray objects.
 How fast are exceptions?
 ------------------------
 
-A try/except block is extremely efficient if no exceptions are raised.  Actually
+A ``try/except`` block is extremely efficient if no exceptions are raised.  Actually
 catching an exception is expensive.  In versions of Python prior to 2.0 it was
 common to use this idiom::
```

Comment:
Like elsewhere, here you can do

```suggestion
A :keyword:`try/except <except>` block is extremely efficient if no exceptions are raised.
Actually catching an exception is expensive.
In versions of Python prior to 2.0, it was common to use this idiom::
```

Reply:
Looking at the [timeit doc](https://github.com/python/cpython/blame/34de67c094776bdc719d87f8810e053a2acc915a/Doc/library/timeit.rst#L311), it just does `` :keyword:`try`/:keyword:`except` `` (even though they link to the same place) and that might be better, in case the links are changed in the future.

Reply:
Indeed I considered that, particularly since as mentioned my #95976 will change that, though I ended up not doing so as the distinction of the separate keywords perhaps not as important here, as the overall block was being referred to (as introduced under the overall `try` section)—though on the other hand, it would be more semantically correct.